.travis.yml

```diff
@@ -19,11 +18,32 @@ env:
     - PYTHON=python3 # python 3.5
     - PYTHON=Conda # not actually a python version, here to test Conda.jl's python
 matrix:
-  exclude:
-    - os: osx
-      env: PYTHON=python3 # I'm not sure how to install Python 3 on Travis OSX
-before_install:
-    - test "x$TRAVIS_OS_NAME" = xosx -o "x$PYTHON" = xConda || (sudo apt-get -qq update && sudo apt-get install -y $PYTHON)
+  include:
+    - &test-macos-py2
+      language: julia
+      os: osx
+      env: PYTHON=python
+      julia: 0.6
+    - &test-macos-py3
+      <<: *test-macos-py2
+      before_install:
+        - brew update
+        - brew upgrade python || echo "Ignoring errors..."
```

Comment:
Strangely, python3 is correctly installed https://travis-ci.org/JuliaPy/PyCall.jl/jobs/444940334#L1356 even though the command fails https://travis-ci.org/JuliaPy/PyCall.jl/jobs/444940334#L1244

Reply:
Actually, we've been using the same trick in PyJulia: https://github.com/JuliaPy/pyjulia/blob/f7a552d2eabbb5045c7182ba056d0867c05830f9/.travis.yml#L49